src/server/dfly_bench.cc

```diff
@@ -647,9 +731,10 @@ void Driver::ParseRESP() {
           CHECK(absl::SimpleAtoi(parts[0], &slot_id));
           vector<string_view> host_port = absl::StrSplit(parts[1], ':');
           CHECK_EQ(host_port.size(), 2u);
-
-          // TODO: to support slot migration.
           LOG_EVERY_T(INFO, 1) << "Moved: " << slot_id << " to " << parts[1];
+          // Slot is moved so update slots information for connecting shard only
+          ClusterSpec shard = FetchCluster(socket_.get(), true);
+          slots_ = shard[0].slots;
```

Comment:
why shard 0? for differrent shards it will be different slots.
this driver got `moved` response due to slot not being owned by this shard but where do you handle the case where a shard (driver) needs to know about slots being added to it.

Reply:
i see it's draft, nm

Reply:
I have modified FetchCluster function, seting second parameter true will filter cluster information and return only shard node (one that is identified with myself) - so we'll only have one vector element

Reply:
ok, it's not possible to see diffs now for FetchCluster. In any case please add a comment explaining why `0` is fine here.